packages/airnode-utilities/src/evm/gas-prices/gas-prices.ts

```diff
@@ -4,6 +4,13 @@ import { go } from '../../promises';
 import { LogsData, logger, PendingLog } from '../../logging';
 import { DEFAULT_RETRY_TIMEOUT_MS, PRIORITY_FEE_IN_WEI, BASE_FEE_MULTIPLIER } from '../../constants';
 
+// This will return the gasLimit with the gasTarget if getGasPrice is called from a place where the fulfillmentGasLimit
+// is available in the config.json and otherwise no gasLimit will be returned for use with other transactions
+// (e.g. withdrawal transactions and airnode-admin CLI/SDK)
+export const getGasLimit = (fulfillmentGasLimit?: number) => {
+  return fulfillmentGasLimit ? { gasLimit: BigNumber.from(fulfillmentGasLimit) } : {};
+};
+
```

Comment:
@aquarat  If you have the chance to check this, I'd appreciate your thoughts on this solution.

We need this mainly for when transactions are submitted from the CLI/SDK from airnode-admin since we don't have the airnode `config.json` there. This should be fine since fulfillment transactions are not submitted from there, right?

Reply:
This approach seems fine to me :)
It's quite elegant.